src/Microsoft.ApplicationInsights.AspNet/Extensions/ApplicationInsightsExtensions.cs

```diff
@@ -48,6 +49,8 @@ public static void AddApplicationInsightsTelemetry(this IServiceCollection servi
             services.AddSingleton<ITelemetryInitializer, WebSessionTelemetryInitializer>();
             services.AddSingleton<ITelemetryInitializer, WebUserTelemetryInitializer>();
 
+            services.AddInstance<IConfiguration>(config);
```

Comment:
A bunch of Unit Tests produce exceptions if I send the first parameter (httpContextAccessor as null). Even if I don't need it on the initializer, as long as the class implements Interface TelemetryInitializerBase it is kind of mandatory to send it not null. 

```
\test\Microsoft.ApplicationInsights.AspNet.Tests\Extensions\Appli
cationInsightsExtensionsTests.cs(52,0): at Microsoft.Extensions.DependencyInjection.ApplicationInsightsExtensionsTests.AddApplicationI
nsightsTelemetry.RegistersExpectedServices(Type serviceType, Type implementationType, ServiceLifetime lifecycle)
      System.ArgumentNullException : Value cannot be null.
      Parameter name: httpContextAccessor
```

```
\test\Microsoft.ApplicationInsights.AspNet.Tests\Extensions\Appli
cationInsightsExtensionsTests.cs(275,0): at Microsoft.Extensions.DependencyInjection.ApplicationInsightsExtensionsTests.AddApplication
InsightsSettings.RegistersTelemetryConfigurationFactoryMethodThatReadsInstrumentationKeyFromSettings()
      System.ArgumentNullException : Value cannot be null.
      Parameter name: httpContextAccessor
```